databases/week3/Meal_query.sql

```diff
@@ -0,0 +1,385 @@
+-- Active: 1719747058156@@127.0.0.1@3306@Meal
+
+CREATE DATABASE Meal;
+
+DEFAULT CHARACTER SET = 'utf8mb4';
+
+use Meal;
+
+CREATE TABLE meal (
+    id INT PRIMARY KEY NOT NULL,
```

Comment:
If your ID is going to be a number that will continuously increment by one, you just set it up as AUTO_INCREMENT. By doing this, you don't need to specify an id each time you insert a new record.

Reply:
Thank you for your feedback. Agreed with your point. I have modified the column with Auto_increment